modules/nf-core/gnu/sort/main.nf

```diff
@@ -0,0 +1,50 @@
+process GNU_SORT {
+    tag "${meta.id}"
+    label "process_low"
+
+    conda "conda-forge::coreutils=9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+    'ubuntu:20.04' }"
+
+    input:
+    tuple val(meta), path(file)
+
+    output:
+    tuple val(meta), file( "*.sorted" )   , emit: sorted
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args        = task.ext.args     ?: ''
+    def output      = task.ext.prefix   ?: "${meta.id}.txt"
```

Comment:
default extension is .txt? should it be all handled by the task.ext.prefix?

Reply:
Referred to below.